docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -20,12 +20,10 @@ This guide will explain how to:
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
-- A host running the `teleport` daemon with the Application Service enabled.
-  Follow the [Getting Started](../getting-started.mdx) or [Connecting
-  Apps](../guides/connecting-apps.mdx) guides to get it running.
+- An AWS EC2 or other instance you can assign a IAM Security Role where
```

Comment:
Should "IAM Security Role" be "IAM role"?

Reply:
Yes, updated to "an IAM role"